docs/guides/cli.md

```diff
@@ -0,0 +1,293 @@
+  -C, --include-context <name>
+                        Context values to display in the comparison table. Use dotted syntax for nested context values.
+  -E, --extra-column <name>
+                        Additional record data to display in the comparison table.
+```
+
+Supposing we have the following records from previous runs:
+
+```json
+// Contents of record1.json:
+{
+  "run": "nnbench-f0822b3c",
+  "context": {
+    "foo": "bar"
+  },
+  "benchmarks": [
+    {
+      "name": "add",
+      "function": "add",
+      "description": "",
+      "date": "2024-11-26T14:04:15",
+      "error_occurred": false,
+      "error_message": "",
+      "parameters": {
+        "a": 50,
+        "b": 25
+      },
+      "value": 75,
+      "time_ns": 917
+    },
+    {
+      "name": "mul",
+      "function": "mul",
+      "description": "",
+      "date": "2024-11-26T14:04:15",
+      "error_occurred": false,
+      "error_message": "",
+      "parameters": {
+        "a": 50,
+        "b": 25
+      },
+      "value": 1250,
+      "time_ns": 625
+    },
+    {
+      "name": "sub",
+      "function": "sub",
+      "description": "",
+      "date": "2024-11-26T14:04:15",
+      "error_occurred": false,
+      "error_message": "",
+      "parameters": {
+        "a": 50,
+        "b": 25
+      },
+      "value": 25,
+      "time_ns": 542
+    }
+  ]
+}
+```
+
+and
+
+```json
+// Contents of record2.json:
+{
+  "run": "nnbench-4f804c24",
+  "context": {
+    "foo": "baz"
+  },
+  "benchmarks": [
+    {
+      "name": "add",
+      "function": "add",
+      "description": "",
+      "date": "2024-11-26T13:16:48",
+      "error_occurred": false,
+      "error_message": "",
+      "parameters": {
+        "a": 50,
+        "b": 25
+      },
+      "value": 75,
+      "time_ns": 2541
+    },
+    {
+      "name": "mul",
+      "function": "mul",
+      "description": "",
+      "date": "2024-11-26T13:16:48",
+      "error_occurred": false,
+      "error_message": "",
+      "parameters": {
+        "a": 50,
+        "b": 25
+      },
+      "value": 1250,
+      "time_ns": 750
+    },
+    {
+      "name": "sub",
+      "function": "sub",
+      "description": "",
+      "date": "2024-11-26T13:16:48",
+      "error_occurred": false,
+      "error_message": "",
+      "parameters": {
+        "a": 50,
+        "b": 25
+      },
+      "value": 25,
+      "time_ns": 584
+    }
+  ]
+},
+```
+
+we can compare them in a table view by running `nnnbench compare record1.json record2.json`:
+
+```commandline
+$ nnbench compare record1.json record2.json
+┏━━━━━━━━━━━━━━━━━━┳━━━━━┳━━━━━━┳━━━━━┓
+┃ Benchmark run    ┃ add ┃ mul  ┃ sub ┃
+┡━━━━━━━━━━━━━━━━━━╇━━━━━╇━━━━━━╇━━━━━┩
+│ nnbench-f0822b3c │ 75  │ 1250 │ 25  │
+│ nnbench-4f804c24 │ 75  │ 1250 │ 25  │
+└──────────────────┴─────┴──────┴─────┘
+```
+
+To include benchmark parameters in the table, use the `-P` switch (you can supply this multiple times to include multiple parameters):
```

Comment:
This refers to the arguments we pass in the benchmarks, right? 

If so, we should half a sentence explaining that as I had to think a bit about it. Someone new to nnbench may not make that connection at all.